autogpt_platform/frontend/.npmrc

```diff
@@ -0,0 +1,2 @@
+# Configure pnpm to save exact versions
+save-exact=true
```

Comment:
- `save-exact=true` ensures that all dependencies are pinned to exact versions (including patch and minor). This helps prevent unexpected version mismatches across environments and CI